src/app.js

```diff
@@ -1,21 +1,43 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 5700;
 
-module.exports = sum;
+const server = http.createServer((request, response) => {
+  const normalizedUrl = new URL(request.url, `http://${request.headers.host}`);
+  let path = normalizedUrl.pathname.split('/');
+
+  if (path[0] !== 'file') {
+    response.statusCode = 404;
```

Comment:
```suggestion
    response.statusCode = 400;
```
If we don't allow a user to make requests without a 'file' we should return a proper error code